contrib/opencensus-ext-azure/opencensus/ext/azure/common/utils.py

```diff
@@ -65,3 +66,20 @@ def timestamp_to_iso_str(timestamp):
 
 def url_to_dependency_name(url):
     return urlparse(url).netloc
+
+
+# Validate UUID format
```

Comment:
Why do we want UUID format link here?
We should have a link to the definition of "what's a valid instrumentation key for Azure Monitor".

Reply:
I don't believe there is an official definition of "what's a valid instrumentation key for Azure Monitor". The links that Mothra had referred to (with the prefix and suffix) do not seem to apply to the ikeys that are sent to Azure Monitor, so we decided that the validation would be to check for UUID. With that being said, I will make note of this in the comments.